src/Sentry/SentryOptions.cs

```diff
@@ -1157,4 +1226,38 @@ public SentryOptions()
                 UriFormat.Unescaped)
         );
     }
+
+    internal void AddIntegration(ISdkIntegration integration)
+    {
+        _integrations.Add(integration);
+    }
+
+    internal void RemoveIntegration<TIntegration>()
+    {
+        // TODO: For backward compatibility, check if it's a default integration type?
```

Comment:
All default integrations are internal, by finding its usages we could spot any removal of default integrations 

```suggestion
    // Note: Not removing default integrations
```

We have usage in Unity (which access internal API) but no default integration there: https://github.com/search?q=repo%3Agetsentry%2Fsentry-unity%20RemoveIntegration&type=code


A comment on the method stating it doesn't apply to default integrations would be good enough I imagine